### ISSUE TITLE
[26421] Separate ticket owner email notification event for Approval Workflow

### ISSUE THREAD

**Issue by iskolares** [DeskDirector/UserVoice#612]
Currently, the approval workflow and corresponding email notification event only emails assigned approvers.

There is no separate event that informs the ticket owner that the ticket is pending approval, or it's rejected or granted.

A separate email event would be good. This way, they can send out emails to ticket owners about the pending approval and possibly use the tokens like who the approvers are the ticket is waiting on

**Comment by Nness** [comment]
You can already achieve that by using status changed email I believe. For example, if status changed to approval required, then the owner of the ticket will receive email.

![image](https://user-images.githubusercontent.com/1712143/188536570-0840d365-68d4-4a45-a637-ee65db25f091.png)
![image](https://user-images.githubusercontent.com/1712143/188536596-86c79835-9cf2-4177-8194-254973f220f5.png)


Of course, display who the approvers are would be nice, but that's some base level information we need to include within the template. I won't close this one but for anyone who check out, you should aware what you can achieve with status changed email.
